### PR TITLE
configs standardization for jrvs

### PR DESCRIPTION
**Pull request by ekmb**

### CODE REVIEW

**Comment by ekmb** [comment]
@purn3ndu please check the default names for the class names in the 3 configs

**Review by purn3ndu — APPROVED** [review]
Looks good to me. Thank you. :)

File: examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml [hunk 1]
```diff
@@ -19,6 +19,9 @@ trainer:
 model:
   nemo_path: null # filename to save the model and associated artifacts to .nemo file
   data_dir: ??? # /path/to/data
+  class_labels:
+    intent_labels_file: intent_labels.csv
+    slot_labels_file: slot_labels.csv
```
Comment (tkornuta-nvidia) [inline examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml:24]: 🥇

File: nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py [hunk 2]
```diff
@@ -260,23 +260,27 @@ def setup_training_data(self, train_data_config: Optional[DictConfig] = None):
         if train_data_config is None:
             train_data_config = self._cfg.train_ds
 
+        if not hasattr(self._cfg, "class_labels") or self._cfg.class_labels is None:
```
Comment (tkornuta-nvidia) [inline nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py:263]: I would suggest to add a comment here that it is a fix for older (pre-1.0.0.b3) configs.

**Review by tkornuta-nvidia — APPROVED** [review]
Awesome, thanks @ekmb this really will help JARVIS (and TLT ;) ) 🥇

File: nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py (hunk 2 above)
Comment (ekmb) [reply nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py:264]: done

**Pull request merged**